### PR TITLE
changed required div's to buttons

### PR DESCRIPTION
**Pull request by rishsingh07**
Using buttons instead of Div tags
Adding Ids to the buttons

### CODE REVIEW

File: src/features/common/Layout/Navbar/Navbar.scss [hunk 1]
```diff
@@ -141,3 +141,11 @@
     }
   }
 }
+
+
+button{
```
Comment (harshvitra) [inline src/features/common/Layout/Navbar/Navbar.scss:146]: Move this to global scss

File: src/features/user/ManageProjects/components/DetailedAnalysis.tsx [hunk 2]
```diff
@@ -581,7 +581,7 @@ export default function DetailedAnalysis({ handleBack, userLang, token, handleNe
                     : null}
 
                 <div className={styles.formField} style={{ marginTop: '48px' }}>
-                    <div className={`${styles.formFieldHalf}`}>
+                    <button id={'backArrowDetailA'} className={`${styles.formFieldHalf}`}>
```
Comment (harshvitra) [inline src/features/user/ManageProjects/components/DetailedAnalysis.tsx:584]: We don't need button here, the Animated button inside is already a button

File: src/features/user/ManageProjects/components/ProjectMedia.tsx [hunk 3]
```diff
@@ -301,7 +301,7 @@ export default function ProjectMedia({ handleBack, token, handleNext, projectDet
           : null}
 
         <div className={styles.formField}>
-          <div className={`${styles.formFieldHalf}`}>
+          <button id={'backArrowProjectM'} className={`${styles.formFieldHalf}`}>
```
Comment (harshvitra) [inline src/features/user/ManageProjects/components/ProjectMedia.tsx:304]: We don't need button here, the Animated button inside is already a button

File: src/features/user/ManageProjects/components/ProjectSites.tsx [hunk 4]
```diff
@@ -334,15 +334,15 @@ export default function ProjectSites({
         ) : null}
 
         <div className={styles.formField}>
-          <div className={`${styles.formFieldHalf}`}>
+          <button id={'backArrowProjectS'} className={`${styles.formFieldHalf}`}>
```
Comment (harshvitra) [inline src/features/user/ManageProjects/components/ProjectSites.tsx:337]: We don't need button here, the Animated button inside is already a button

File: src/features/user/UserProfile/components/RedeemModal.tsx [hunk 5]
```diff
@@ -153,7 +153,7 @@ export default function RedeemModal({
           <>
             <div className={styles.modalFinal}>
               <div className={styles.header}>
-                <div onClick={() => closeRedeem()} className={styles.headerCloseIcon}>
+                <button id={'closeRedeemM'} onClick={() => closeRedeem()} className={styles.headerCloseIcon}>
```
Comment (harshvitra) [inline src/features/user/UserProfile/components/RedeemModal.tsx:156]: Close this button tag

File: src/features/user/UserProfile/components/EditProfileModal.tsx [hunk 6]
```diff
@@ -228,9 +228,9 @@ export default function EditProfileModal({
                       <CameraWhite />
                     </div>
```
Comment (harshvitra) [inline src/features/user/UserProfile/components/EditProfileModal.tsx:229]: Close this button

File: src/features/user/UserProfile/components/EditProfileModal.tsx [hunk 7]
```diff
@@ -191,7 +191,7 @@ export default function EditProfileModal({
         <div className={styles.modal}>
           <div>
             <div className={styles.headerDiv}>
-              <div
+              <button id={'backButtonEditP'}
```
Comment (harshvitra) [inline src/features/user/UserProfile/components/EditProfileModal.tsx:194]: Close this button

File: src/features/user/ManageProjects/components/ProjectSpending.tsx [hunk 8]
```diff
@@ -280,7 +280,7 @@ export default function ProjectSpending({ handleBack, token, handleNext, userLan
                     : null}
 
                 <div className={styles.formField}>
-                    <div className={`${styles.formFieldHalf}`}>
+                    <button  id={'backArrowProjectSpend'} className={`${styles.formFieldHalf}`}>
```
Comment (harshvitra) [inline src/features/user/ManageProjects/components/ProjectSpending.tsx:283]: We don't need button here, the Animated button inside is already a button

**Review by harshvitra — CHANGES_REQUESTED** [review]
Fix the above comments

File: src/features/user/UserProfile/components/AddTargetModal.tsx [hunk 9]
```diff
@@ -106,16 +106,16 @@ export default function AddTargetModal({
...
             </div>
           ) : (
-            <div
+            <button id={'AddTargetCont'}
               className={styles.continueButton}
               onClick={() => handleSubmit(apiCallChangeTarget())}
             >
               {isLoadingForm ? (
-                <div className={spinnerStyle.spinner}></div>
+                <button className={spinnerStyle.spinner}></div>
```
Comment (harshvitra) [inline src/features/user/UserProfile/components/AddTargetModal.tsx:114]: This is not a button

File: src/features/user/UserProfile/components/EditProfileModal.tsx [hunk 10]
```diff
@@ -228,9 +228,9 @@ export default function EditProfileModal({
                       <CameraWhite />
                     </div>
                   ) : (
-                    <div className={styles.noProfilePic}>
+                    <button id={'userCamera'} className={styles.noProfilePic}>
```
Comment (harshvitra) [inline src/features/user/UserProfile/components/EditProfileModal.tsx:231]: This is not a button

**Review by harshvitra — CHANGES_REQUESTED** [review]

**Comment by norbertschuler** [comment]
Build fails with:

```
13:06:52.112 | Failed to compile.
-- | --
13:06:52.112 | ./src/features/user/UserProfile/components/AddTargetModal.tsx:114:57
13:06:52.112 | Syntax error: Expected corresponding JSX closing tag for <button>
13:06:52.112 | 112 \|             >
13:06:52.112 | 113 \|               {isLoadingForm ? (
13:06:52.112 | > 114 \|                 <button className={spinnerStyle.spinner}></div>
13:06:52.112 | \|                                                          ^
13:06:52.112 | 115 \|               ) : (
13:06:52.112 | 116 \|                 t('target:targetSave')
13:06:52.112 | 117 \|               )}


```

File: src/features/user/UserProfile/components/AddTargetModal.tsx (hunk 9 above)
Comment (norbertschuler) [reply src/features/user/UserProfile/components/AddTargetModal.tsx:114]: - [ ] Build fails with:

```
13:06:52.112 | Failed to compile.
-- | --
13:06:52.112 | ./src/features/user/UserProfile/components/AddTargetModal.tsx:114:57
13:06:52.112 | Syntax error: Expected corresponding JSX closing tag for <button>
13:06:52.112 | 112 \|             >
13:06:52.112 | 113 \|               {isLoadingForm ? (
13:06:52.112 | > 114 \|                 <button className={spinnerStyle.spinner}></div>
13:06:52.112 | \|                                                          ^
13:06:52.112 | 115 \|               ) : (
13:06:52.112 | 116 \|                 t('target:targetSave')
13:06:52.112 | 117 \|               )}
```

**Comment by sagararyal** [comment]
https://html.spec.whatwg.org/multipage/dom.html#element-attrdef-global-id

These IDs should be unique in the page. Which are not for example the button below:

Screen Shot 2021-01-26 at 11 32 35 PM


Something you can do is use the project guid to generate the id

for the single project
<div id="proj_eKBbIt7Bzavu9o7xzCAqjS2t" >

and for button 
<button id="don_proj_eKBbIt7Bzavu9o7xzCAqjS2t">


Menu buttons should be wrapped inside nav
<nav> 
<button> </button>
<button> </button>
<button> </button>
</nav>
Screen Shot 2021-01-26 at 11 37 35 PM

**Comment by harshvitra** [comment]
> Menu buttons should be wrapped inside nav

Let's create a different issue and branch for this. This branch is already too big.

**Comment by sagararyal** [comment]
@rishsingh07 
Screen Shot 2021-01-27 at 10 49 18 PM
these needs to be unique as well; and if possible make the keys shorter.

**Pull request merged**